libraries/core-react/src/components/Card/Card.tsx

```diff
@@ -28,24 +27,35 @@ const StyledCard = styled.div<StyledCardProps>`
   grid-auto-columns: auto;
   align-items: center;
   align-content: start;
-  border-radius: ${shape.borderRadius};
-  min-height: ${shape.minHeight};
   cursor: ${({ cursor }) => cursor};
-
-  ${spacingsTemplate(spacings)}
+  ${bordersTemplate(primary.border)};
+  ${spacingsTemplate(primary.spacings)}
 `
 
+type backgroundVariants = { variant: string; backgroundColor: string }[]
+
+const backgroundVariants: backgroundVariants = [
+  { variant: 'default', backgroundColor: primary.background },
+  { variant: 'info', backgroundColor: info.background },
+  { variant: 'warning', backgroundColor: warning.background },
+  { variant: 'danger', backgroundColor: danger.background },
+]
+
 export const Card = forwardRef<HTMLDivElement, CardProps>(function Card(
   { children, className, variant = 'default', onClick, ...rest },
   ref,
 ) {
+  const cardVariant = backgroundVariants.find((elem) => {
+    return elem.variant === variant
+  })
+
   const cursor = onClick ? 'pointer' : 'default'
 
   const props = {
     ...rest,
     className,
     ref,
-    background: tokens.background[variant],
+    background: cardVariant && cardVariant.backgroundColor,
```

Comment:
This feels abit of an overkill for a simple object lookup?

My suggestion would be something like this maybe?
``` 
const lookupKey = variant === 'default' ? 'primary' : variant

const token = tokens[lookupKey]

const props = {
...
background: token.background,
...
}

```